Fitness/Networking/NetworkManager.swift

```diff
@@ -15,7 +15,7 @@ enum APIEnvironment {
 }
 
 struct NetworkManager {
-    internal let apollo = ApolloClient(url: URL(string: "http://localhost:5000/")!)
+    internal let apollo = ApolloClient(url: URL(string: "http://178.128.135.122/")!)
```

Comment:
can we point this at the DNS record, so `http://uplift-backend.cornellappdev.com`